safekeeper/src/timelines_global_map.rs

```diff
@@ -296,15 +296,21 @@ impl GlobalTimelines {
         }
     }
 
-    /// Returns all timelines. This is used for background timeline processes.
+    /// Returns all timelines. Note that it *includes* cancelled (deleted)
+    /// timelines.
     pub fn get_all() -> Vec<Arc<Timeline>> {
+        let global_lock = TIMELINES_STATE.lock().unwrap();
+        global_lock.timelines.values().cloned().collect()
```

Comment:
I don't think we should return deleted timelines from `get_all()`. They're not very useful and ideally shouldn't be used for anything.